recipes-core/custom-nftables-rules/files/nftables-production.conf

```diff
@@ -0,0 +1,40 @@
+#!/usr/sbin/nft -f
+
+flush ruleset
+
+define BUILDER_RPC_SRC = 0.0.0.0  #TODO: Replace with actual IP address
+define LOGS_COLLECTOR_SRC = 1.1.1.1 #TODO: Replace with actual IP address
+define LOGS_COLLECTOR_PORT = 1234 #TODO: Replace with actual port
+
+table inet filter {
+    chain input {
+        type filter hook input priority 0; policy drop
+
+        # Allow established/related connections
+        ct state established,related accept
+
+        # Production mode rules
+        tcp dport 8551 accept comment "Consensus Client Engine API"
```

Comment:
Hm, should we be accepting requests to CL API apart from p2p?
I think Lighthouse listens at 9000 by default, should probably be listened for and accepted

Reply:
yeah, you are right but I was following the table in Angela's doc [here](https://www.notion.so/flashbots/BoB-external-tasks-1126b4a0d876804f924dea7b8ce1cfbc?pvs=4#11e6b4a0d876804195becec99d967cfd).
I guess, this is specific to the deployment of princess stuff within podman. We need more context sync with Angela on that doc

Reply:
Spoke with searcher, they prefer to run CL on their machine too. So let's remove 8551, open 9000 on our trusted CL node outside, and forward to 9000 on the searcher's machine.